### DISCUSSION TITLE
Check for all cryptic (hidden GCI) inborn vs acquired assumptions

### DISCUSSION THREAD

**Discussion by cmungall** [monarch-initiative/mondo#4670] [Bigger project]
There are many cases where "disease of X metabolism" is classified (hidden GCI) as inborn/genetic when in fact acquired cases exist. We are now finding these as we add disjointness axioms. Most have been discovered. The fix here is to either

 1. Rename the class so that "inborn" is in the label (and text def and owl def are consistent)
 2. OR keep the class etiology-generic and remove the hidden GCI, and potentially create a new subclass(es) for acquired vs inborn

Most of these have already been caught but some remain. These gradually become uncovered when we axiomatize previously logically silent classes like 'rare acquired hemolytic anemia'

E.g

![image](https://user-images.githubusercontent.com/50745/38770087-1c06d562-3fc2-11e8-969e-7f045cc03b60.png)

(reported here: https://github.com/Orphanet/ORDO/issues/4)

This one is nuanced. The disorder is caused by a somatic mutation in PIGA. It is not inborn, but is genetic

one approach is to generalize the whole hierarchy:

```
   is_a MONDO:0019052 ! inborn errors of metabolism
    is_a MONDO:0019214 ! inborn carbohydrate metabolic disorder
     is_a MONDO:0015286 ! congenital disorder of glycosylation
      is_a MONDO:0017748 ! disorder of glycosphingolipid and glycosylphosphatidylinositol anchor glycosylation *** 
```

another is to parallelize/shadow

**Comment by cmungall** [comment]
Another example, STAT3-related early-onset multisystem autoimmune disease

![image](https://user-images.githubusercontent.com/50745/38771201-aa9c7492-3fd3-11e8-96b6-7b718316ac30.png)

https://www.omim.org/entry/615952

the disease is caused by germline mutation of STAT3, causing autoimmune disease, assumption that autoimmune hemolytic anemia is acquired seems false

**Comment by pnrobinson** [comment]
This phrase is misleading "It is not inborn, but is genetic"
The word "inborn" is not precisely defined, but one definition puts is as "present from birth" (https://www.merriam-webster.com/dictionary/inborn), and therefore, a somatic mutation that affects the egg or the sperm is 9 months prior to that event and thus a somatic mutation is definitely inborn.
The word "genetic" can also have multiple meanings, and I doubt that anybody in the community would be able to agree on a precise way of classifiying Mendelian, oligogenic, polygenic, imprinting, de novo mutation, mosiac de novo mutation etc etc.
Therefore, it is not useful to use either of these words as delineators, this will only wind up confusing people in the community. 
Even the word inherited is potentially confusing. If a parent does not have the disease in question but that parent produce a gamete that contains a de novo mutation, then still in some sense the anlage for disease was inherited by the child from the parent.

**Comment by pnrobinson** [comment]
I think that it will be important to first decide on the overall strategy ("philosophy") of classification in MONDO, and then try to fix the hierarchy. I am concerned that doing ad hoc fixes will not lead to a coherent whole, and I am also concerned that the English words that we are using to make decisions (inborn, hereditary, congenital...) or so poorly defined and/or in many cases there is an agreed upon meaning in the community (for instance, if you have an acidosis related to a congenital infection, then logically speaking you have an inborn error of metabolism, but the community uses "inborn error of metabolism" to refer to Mendelian diseases caused by mutation in genes for metabolic enzymes).

**Comment by cmungall** [comment]
not sure why this was converted to discussion the original issue was an actionable piece of QC (the exact terminology isn't relevant here)

**Reply by nicolevasilevsky** [reply]
my bad, I recreated a ticket here: https://github.com/monarch-initiative/mondo/issues/4672

**Comment by cmungall** [comment]
make a separate issue to rename all "inborn X" terms if we feel that is necessary
